doc/whats_new.rst

```diff
@@ -40,6 +40,9 @@ Changelog
   returned for cases when there should be no valid matches, by `George Dang`_
   in :gh:`739`
 
+- Added check for invalid Axes object in :func:`~hnn_core/viz/plot_cells` 
+  function, by `Abdul Samad Siddiqui`_ in :gh:`744`
```

Comment:
You'll also want to link your name to your GH account at the [bottom of the file](https://github.com/jonescompneurolab/hnn-core/blob/54bd27805198f8c345efe307b2860ad271f1dc3f/doc/whats_new.rst?plain=1#L469).